### PR TITLE
integrated Processable function into main.m 

### PR DESCRIPTION
**Pull request by clivethe14**
main.m can now handle 160 images for each case and have created 2 folders to test 2 different cases.

### CODE REVIEW

File: code/AverageCartilageThickness.m [hunk 1]
```diff
@@ -1,5 +1,7 @@
 function averageThickness = AverageCartilageThickness(image)
     BinaryImage = image;
+    
+    BinaryImage(1:90,:)= 255;
```
Comment (clivethe14) [inline code/AverageCartilageThickness.m:4]: more padding added above femur for code to recognize femur correctly based on larger area of femur.

File: code/Processable.asv
Comment (mj34564n) [inline code/Processable.asv:1]: ok with either this method or Tarun's method of calculating area before

**Review by mj34564n — COMMENTED** [review]
yes

**Review by mj34564n — APPROVED** [review]

**Comment by clivethe14** [comment]
approved

**Pull request merged**